BE/src/main/java/com/mj_eno/sidedish/web/DishController.java

```diff
@@ -0,0 +1,57 @@
+package com.mj_eno.sidedish.web;
+
+import com.mj_eno.sidedish.service.DishService;
+import com.mj_eno.sidedish.web.dto.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@CrossOrigin("*")
+public class DishController {
+
+    public final DishService dishService;
+    private final Logger logger = LoggerFactory.getLogger(DishController.class);
+
+    public DishController(DishService dishService) {
+        this.dishService = dishService;
+    }
+
+    @GetMapping("/best")
+    public List<BestDishResponseDTO> viewAllBestDishes() {
+        logger.info("모든 best 메뉴 요청");
+        return dishService.findAllBestDish();
+    }
+
+    @GetMapping("/best/{categoryId}")
+    public BestDishResponseDTO viewBestDishesByCategory(@PathVariable Long categoryId) {
+        logger.info(categoryId + "번 카테고리의 best 메뉴 요청");
```

Comment:
`logger` 에서는 문자열 템플릿을 사용하는 것이 좋다고 리뷰를 충분히 드렸었던 것 같습니다.

```suggestion
        logger.info("{}번 카테고리의 best 메뉴 요청", categoryId);

Reply:
이 부분도 반성합니다... 더 신경 써야겠네요!

Reply:
코멘트에 따라 템플릿에 맞춰서 수정했습니다~